asv/plugin_manager.py

```diff
@@ -30,7 +30,7 @@ def load_plugins(self, package):
                 self.init_plugin(mod)
                 self._plugins.append(mod)
             except ModuleNotFoundError as err:
-                log.error(f"Couldn't load {name} because\n{err}")
+                log.debug(f"Couldn't load {name} because\n{err}")
                 continue  # Couldn't find mamba
```

Comment:
This code path can only be hit if package == mamba? Maybe the error is fine and the problem is in the code that is calling this function?

Reply:
I reworked this to make it clearer. It starts to crop up with the `mamba` plugin, since that needs `libmambapy` and imports it. The error will also be triggered for `virtualenv` (but not `conda` since that will raise an `OSError` if it isn't found).